portfolio/src/main/webapp/script.js

```diff
@@ -30,7 +30,9 @@ function getRandomFactAboutRoss() {
 
   // Pick a random fact.
   const fact = facts[Math.floor(Math.random() * facts.length)];
-
+  if(fact === 'I have been to 4 continents!' || fact === 'I have been to more than 10 countries!') {
```

Comment:
add new variable before query on array 'facts' that indicates the index. Then check that index here instead of string.